.devcontainer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM docker.io/library/fedora:37
+
+ENV CC=clang
+
+ENV WASI_SDK_VERSION=19
+ENV WASI_SDK_PATH=/opt/wasi-sdk
+
+ENV WASMTIME_HOME=/opt/wasmtime
+ENV WASMTIME_VERSION=7.0.0
+ENV WASMTIME_CPU_ARCH=x86_64
+
+RUN dnf -y --nodocs install git clang xz python3-blurb dnf-plugins-core && \
```

Comment:
I realize I am late to the party, but you might save space by installing git-core instead of git, which should avoid Perl (unless something else brings it in).
(`dnf builddep python3` will bring in git-core as well, because we use git to apply patches, but that is probably not explicit enough.)

Reply:
Invoking dnf with `--setopt=install_weak_deps=False` will also save space. Let me try sending a PR and see if it works.

Reply:
https://github.com/python/cpython/pull/103283